extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/OtTracerPropagator.java

```diff
@@ -64,8 +65,9 @@ public static OtTracerPropagator getInstance() {
   }
 
   @Override
-  public <C> Context extract(Context context, C carrier, Getter<C> getter) {
+  public <C> Context extract(Context context, @Nullable C carrier, Getter<C> getter) {
     if (context == null || getter == null) {
+      // TODO Other propagators throw exceptions here
```

Comment:
This should probably fixed -- either remove the check and let an NPE occur "naturally" or use Objects.requireNonNull. Or we could change all propagators to do something like here (best with logging though).

In any case, this should be a follow-up.